design/detail-list/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { classNames } from '@oacore/design/lib/utils'
+
+import styles from './styles.css'
+
+const DetailList = React.memo(({ children, className, ...passProps }) => (
```

Comment:
`Details` and `Details.Item`? I am not sure

Reply:
Yep maybe. If you want you can rename it in your PR. :)